src/EditorFeatures/CSharpTest/AddAccessibilityModifiers/AddAccessibilityModifiersTests.cs

```diff
@@ -165,5 +171,164 @@ namespace Test
     internal readonly struct S1 { }
 }");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddAccessibilityModifiers)]
+        public async Task TestAllConstructsWithOmit()
+        {
+            await TestInRegularAndScriptAsync(
+@"
+namespace Outer
+{
+    namespace Inner1.Inner2
+    {
+        internal class {|FixAllInDocument:C|}
+        {
+            private class NestedClass { }
+
+            private struct NestedStruct { }
+
+            private int f1;
+            private int f2, f3;
+            public int f4;
+
+            private event Action e1, e2;
+            public event Action e3;
+
+            private event Action e4 { add { } remove { } }
+            public event Action e5 { add { } remove { } }
+            event Action I.e6 { add { } remote { } }
+
+            static C() { }
+
```

Comment:
❗️ This new line is correctly preserved by the fix, but is not present in the expected test result, leading to a test failure. Either remove this blank line or add one in the expected result so the input/output are consistent.

Reply:
fixed, thanks!